js/everything.js

```diff
@@ -60,6 +60,11 @@ var plyrPlayer;
 var youTubeDataApiKey = "AIzaSyCxVxsC5k46b8I-CLXlF3cZHjpiqP_myVk";
 var currentVideoID;
 
+// global playlist, this is populated with an ajax call
+var playList = [];
+var autoplayState = false;
+var videoMetaData = {};
```

Comment:
super nit: Metadata is 1 word, I'd like to see `videoMetadata` everywhere. Don't feel obligated to do this one